helm/cert-manager-app/charts/giantswarm-cluster-issuer/templates/job.yaml

```diff
@@ -26,12 +26,10 @@ spec:
         volumeMounts:
         - name: {{ .Values.name }}
           mountPath: /data/
+        resources: {{- toYaml .Values.resources | nindent 8 }}
       volumes:
-      - configMap:
+      - name: {{ .Values.name }}
+        configMap:
           name: {{ .Values.name }}
-          namespace: {{ .Release.Namespace }}
```

Comment:
Removed this invalid field ref which seems to be ignored by helm 2 and k8s, but helm 3 was complaining